packages/eslint-plugin/__tests__/stylex-margin-padding.js

```diff
@@ -50,44 +70,17 @@ eslintTester.run('stylex-sort-keys', rule.default, {
         import stylex from 'stylex';
         const styles = stylex.create({
           main: {
-            marginInlineTop: '10px',
+            marginTop: '10px',
             marginInlineEnd: '12px',
-            marginInlineBottom: '13px',
+            marginBottom: '13px',
             marginInlineStart: '14px',
```

Comment:
Auto-expanding into a mixture of logical and physical properties is probably not right. `margin: 1px 2px 3px 4px` is not treated as logical by browsers, so we should probably auto-expand to physical

Reply:
these are the agreed upon expansions after syncing with @nmn, because of older browser compatibility. letting @nmn chip in with any thoughts